{{cookiecutter.project_slug}}/backend/{{cookiecutter.project_slug}}/challenge/apps.py

```diff
@@ -0,0 +1,16 @@
+# demo/challenge/apps.py
```

Comment:
You have hardcoded the `demo` python module here and below, make sure to use the templated variable for the project slug since folks could create a project with any name.